test-end-to-end/webapp-cypress/cypress/e2e/004_003_user_add_private_comment.cy.js

```diff
@@ -0,0 +1,81 @@
+import setDate from '../support/setDate'; //https://github.com/cypress-io/cypress/issues/1366#issuecomment-437878862
+describe('004_003_user_add_private_comment', () => {
+it('Start Ustad Test Server ', () => {
+ // Start Test Server
+  cy.ustadStartTestServer()
+})
+
+it('Admin add a course and Members', () => {
+ // Admin user login
+  cy.ustadClearDbAndLogin('admin','testpass')
+ // Add a new course
+  cy.ustadAddCourse('004_003')
+ // Add module block
+ // cy.contains('button','Edit').click()
+ // cy.ustadAddModuleBlock('module 1')
+ //Add a teacher
+  cy.contains("button","Members").click()
+  cy.contains("span","Add a teacher").click()
+  cy.ustadAddNewPerson('Teacher','1','Female')
+ // Add account for teacher
+  cy.contains("Teacher 1").click()
+  cy.contains('View profile').click()
+  cy.ustadCreateUserAccount('teacher1','test1234')
+ //Add a student1
+  cy.contains("span","Add a student").click()
+  cy.ustadAddNewPerson('Student','1','Male')
+  cy.contains("button","Members").should('be.visible')
+ //Add account for student1
+  cy.contains("Student 1").click()
+  cy.contains('View profile').click()
+  cy.ustadCreateUserAccount('student1','test1234')
+ //Add a student2
+  cy.contains("span","Add a student").click()
+  cy.ustadAddNewPerson('Student','2','Male')
+  cy.contains("button","Members").should('be.visible')
+ //Add account for student1
+  cy.contains("Student 2").click()
+  cy.contains('View profile').click()
+  cy.ustadCreateUserAccount('student2','test1234')
+})
+
+it('Teacher add assignment and course comment', () => {
+  cy.ustadClearDbAndLogin('teacher1','test1234')
+
+ // Add Assignment block
+  cy.contains("Courses").click()
+  cy.contains("004_003").click()
+  cy.contains("button","Members").click()  // This is a temporary command to make sure member list is loaded
+  cy.contains("button","Course").click()
+  cy.contains("button","Edit").click()
+  cy.contains("Add block").click()
+  cy.contains("Assignment").click()
+  cy.get('input[id="title"]').type("Assignment 1")
+  cy.get('div[data-placeholder="Description"]').type("this is a simple assignment")
+  cy.contains("button","Done").should('be.visible')
+  cy.contains("button","Done").click()
+  cy.contains("button","Save").should('be.visible')
+  cy.contains("button","Save").click()
+  cy.contains("button","Members").should('be.visible')
+  cy.contains("Assignment 1").click()
+   cy.contains('Submissions').click()
+  cy.contains('Student 1').click()
+  cy.get('#private_comment_textfield').click()
+  cy.get('#private_comment_textfield').type("comment1")
+  cy.get('svg[data-testid="SendIcon"]').click()
```

Comment:
This button should have an id - should be private_comment_textfield_submit .

Reply:
Send icon does not have an id, checked. Can you please add ID to this icon.